packages/grid/_modules_/grid/components/panel/Panel.tsx

```diff
@@ -43,6 +44,23 @@ export function Panel(props: PanelProps) {
   const classes = useStyles();
   const apiRef = React.useContext(ApiContext);
 
+  const getPopperModifiers = (): any => {
```

Comment:
Popper is also used in GridMenu. So could we create a wrapper component around Popper to encapsulate this logic?

Reply:
Does GridMenu need to disable the flip behavior?